internal/monitors/postgresql/monitor.go

```diff
@@ -75,7 +78,12 @@ func (m *Monitor) Configure(conf *Config) error {
 	m.ctx, m.cancel = context.WithCancel(context.Background())
 
 	var err error
```

Comment:
`err` variable isn't used anymore in this function. (All usages are now with `:=`).